test_runner/src/main/kotlin/ftl/client/google/run/android/GcAndroidTestMatrix.kt

```diff
@@ -73,18 +70,11 @@ private fun createAndroidTestMatrix(
     }.getOrElse { e -> throw FlankGeneralError(e) }
 }
 
-fun TestMatrixAndroid.Config.clientInfo(matrix: TestMatrixAndroid.Type): ClientInfo {
-    return if (matrix is TestMatrixAndroid.Type.Instrumentation && matrix.clientDetails.isNotEmpty()) {
```

Comment:
@pawelpasterz I am sure that there were changes done here for a fix by @asadsalman. Can we confirm that it wont be effected by this change as it seems like a revert to the previous version of the code.

Reply:
You are right - this will revert those changes. But this is intentional, previous implementation merges client details (from main and additional test pair). All the other, 'additional' configs overwrite 'main' configs. I think we should be consistent here. Saying that merging is not the best option and `clientDetails` should be overwritten as well.